build_irods_packages.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
+
+# Compile the GenQuery2 project.
+git clone https://github.com/irods/irods_api_plugin_genquery2
+mkdir /_build_genquery2
+cd /_build_genquery2
+cmake -GNinja -DIRODS_ENABLE_430_COMPATIBILITY=NO /irods_api_plugin_genquery2
+ninja package
+apt install -y ./*.deb
+cp *.deb /packages_output
```

Comment:
switch these two... so builds, then copies (same as below)... then... installs.

Reply:
I'm just now seeing this. I'm not sure if it still applies now.

Please see updated script.

Reply:
yep, no worries.   resolved.